src/test/java/de/szut/lf8_project/lf8/project/CreateIT.java

```diff
@@ -0,0 +1,154 @@
+package de.szut.lf8_project.lf8.project;
+
+import de.szut.lf8_project.testcontainers.AbstractIntegrationTest;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.springframework.http.MediaType;
+import org.springframework.security.test.context.support.WithMockUser;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import static org.hamcrest.Matchers.hasItem;
+import static org.hamcrest.Matchers.is;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.delete;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+public class CreateIT extends AbstractIntegrationTest {
+    @Test
+    public void authorization() throws Exception {
+        this.mockMvc.perform(post("/lf8/project/create"))
+                .andExpect(status().isUnauthorized());
+    }
+
+    @Test
+    @WithMockUser(roles = "user")
+    public void create() throws Exception {
+        String content = """
+                {
+                    "customerId": 1,
+                    "responsibleEmployeeId": 2,
+                    "responsibleCustomerEmployeeId": 3,
+                    "description": "Test description"
+                }
+                """;
+
+        this.mockMvc.perform(
+                post("/lf8/project/create")
+                        .content(content)
+                        .contentType(MediaType.APPLICATION_JSON)
+                        .header("Authorization", token))
+                .andExpect(status().isCreated())
+                .andExpect(jsonPath("id").exists())
+                .andExpect(jsonPath("customerId", is(1)))
+                .andExpect(jsonPath("responsibleEmployeeId", is(2)))
+                .andExpect(jsonPath("responsibleCustomerEmployeeId", is(3)))
+                .andExpect(jsonPath("description", is("Test description")))
+                .andReturn()
+                .getResponse()
+                .getContentAsString();
+    }
+
+    //TODO: Einen Test der das Enddatum vor das Begindatum packt
```

Comment:
This should probably get done before the merge

Reply:
Ja hab ich auch drüber nachgedacht. Aber ich hatte bis jetzt keine Lust :Kekw:

Reply:
bruhhhhhhh

Reply:
Ah wait ne das ist schon gemacht weiter unten. Ich habe den TODO einfach nicht entfernt c: